lesson07/assignment/src/views/accordion.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div class="accordion">
+       <section class="main_accordion">
+            <div id="accordion">
+                <div v-for="poster in posters" class="card">
+                    <div class="card-header">
+                        <a v-bind:href="'#' + poster.div_id" data-parent="#accordion"
+                            data-toggle="collapse">
+                            <img v-bind:src="poster.img_link" alt="">
+                        </a>
+                    </div>
+                    <div v-bind:id="poster.div_id" class="collapse">
+                        <div class="card-body">
+                            <h3 class="display-4" data-toggle="modal" v-bind:data-target="'.' + poster.div_class">{{poster.movie_name}}</h3>
+                            <div v-bind:class="'modal fade ' + poster.div_class" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel" aria-hidden="true">
+                                <div class="modal-dialog modal-lg">
+                                    <div class="modal-content">
+                                     {{poster.modal_line_1}}<br>
+                                     {{poster.modal_line_2}}<br>
+                                     {{poster.modal_line_3}}<br>
+                                     {{poster.modal_line_4}}<br>
+                                     {{poster.modal_line_5}}<br>
+                                     {{poster.modal_line_6}}<br>
+                                     {{poster.modal_line_7}}<br>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="display2 comments" alt="To add a comment refresh the page">
+                <p v-if="show" ><span>{{nickname}}: </span> {{comment_text}}</p>
+                <div v-else="show"class="comment_form">
```

Comment:
For else, you can just omit everything after v-else, so we don't need an ="show" since Vue knows the `v-else` follows the `v-if`